presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -862,11 +874,21 @@ The following properties can be used to configure the :doc:`./web-interface`.
     a new random value, which in turn invalidates the session of any currently
     logged in Web UI user.
 
-web-ui.session-timeout
-^^^^^^^^^^^^^^^^^^^^^^
+``web-ui.session-timeout``
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
     * **Type:** ``duration``
     * **Default value:** ``1 day``
 
     The duration how long a user can be logged into the Web UI, before the
     session times out, which forces an automatic log-out.
+
+``web-ui.user``
+^^^^^^^^^^^^^^^
+
+    * **Type:** ``string``
+    * **Default value:**
+
+    The username automatically used for authentication to the UI with the
```

Comment:
maybe `s/to the UI/in the Web UI`.. should definitely be `Web UI to match the other references.

Reply:
Fixed